### DISCUSSION TITLE
86Box v3.1 Release date

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#1954] [General]
Alright. So, I was excited about v3.1 beta build, and now, I've been waiting for a release version to come. When will it be released?

Also, I'm sorry if this discussion is not welcomed here...

**Comment by ghost** [comment]
Early 2022

**Comment by waltje** [comment]
Christmas.  Its just not clear which year :)--Sent via my Samsung Galaxy Note® 4 4G LTE

-------- Original message --------
From: BedrockDotPng ***@***.***> 
Date: 12/23/21  1:24 AM  (GMT-05:00) 
To: 86Box/86Box ***@***.***> 
Cc: Subscribed ***@***.***> 
Subject: [86Box/86Box] 86Box v3.1 Release date (Discussion #1954) 


Alright. So, I was excited about v3.1 beta build, and now, I've been waiting for a release version to come. When will it be released?
Also, I'm sorry if this discussion is not welcomed here...

—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/86Box/86Box/discussions/1954",
"url": "https://github.com/86Box/86Box/discussions/1954",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by OBattler** [comment]
We have decided to switch to a rolling release model, so 3.1 is going to be released in January (or maybe even sooner, since I really don't want to be doing this for new year) with whatever is currently in.

**Reply by ghost** [reply]
> We have decided to switch to a rolling release model, so 3.1 is going to be released in January (or maybe even sooner, since I really don't want to be doing this for new year) with whatever is currently in.

Alright. We'll wait.

**Reply by ghost** [reply]
WHAT?! I thought it would be at January 2022, but no... So I am waiting for v3.2.

**Reply by OBattler** [reply]
3.1 is already out, we released it right before Christmas.

**Reply by jriwanek** [reply]
I'm trying to work out if you're a troll or not. As has been clearly stated, 86box is on a rolling release now.

**Comment by jriwanek** [comment]
To explain further:

we provide releases on our build server (https://ci.86box.net/job/86Box/) for every change which is added, "stable" releases are just a snapshot at a given moment in time (Currently roughly monthly), and don't really imply any kind of real release persay, just that users seem to expect a number to increase, changes in any given release are basically "what was done at the time of that release".

as a further addendum, and hopefully you now understand that a given "release" isn't going to be of particular interest beyond a number increasing compared to what's available on the build server, I'd also like to point out that every time someone asks us when x version will release, we have to stop working to answer that question, and therefore there will be less available when we do provide it.

I hope this answers any further questions you may have.
